server/data/orchestrationApiClient.ts

```diff
@@ -352,4 +358,15 @@ export default class OrchestrationApiClient {
     const mainContactId = mainContact?.contactId ?? null
     return { visitContact, mainContactId }
   }
+
+  // TODO remove when property rename API change deployed
+  private remapAlertsDates(alerts: Alert[]): Alert[] {
+    return alerts.map((alert: AlertDtoRaw) => {
+      return {
+        ...alert,
+        startDate: alert.startDate ?? (alert.dateCreated || null),
+        expiryDate: alert.expiryDate ?? (alert.dateExpires || null),
```

Comment:
Handle incoming old and new API values but return the new ones. E.g. prefer `expiryDate` but if not set use `dateExpires` - or failing that, `null`